backend/lcfs/tests/conftest.py

```diff
@@ -32,26 +34,23 @@ def pagination_request_schema():
 
 
 @pytest.fixture
-def mock_user_profile():
-    def _create_mock_user_profile(
-        role="GOVERNMENT",
-        organization_id=1,
-        email="john.doe@example.com",
-        user_profile_id=1,
-    ):
-        class MockUserProfile:
-            def __init__(self):
-                self.user_profile_id = user_profile_id
-                self.first_name = "John"
-                self.last_name = "Doe"
-                self.keycloak_username = "johndoe"
-                self.organization_id = organization_id
-                self.email = email
-                self.role_names = [role]
-
-            def get_role_names(self):
-                return self.role_names
+def mock_user_profile(
+    role=RoleEnum.GOVERNMENT,
```

Comment:
I think we have a few more places to update with the enum checks. If you run a find all on the all caps role names, see if you can find any others to fix.

Reply:
Done! Looks like the were ~100 or so but I think I got them.